src/objective-c/GRPCClient/private/GRPCWrappedCall.m

```diff
@@ -274,6 +276,10 @@ - (void)startBatchWithOperations:(NSArray *)operations {
 }
 
 - (void)startBatchWithOperations:(NSArray *)operations errorHandler:(void (^)())errorHandler {
+#ifdef GRPC_TEST_OBJC
```

Comment:
add a comment that explains the reason behind not enabling this code when not testing.